tensorboard/plugins/core/core_plugin.py

```diff
@@ -285,6 +292,31 @@ def _serve_experiment_runs(self, request):
         results = []
         return http_util.Respond(request, results, "application/json")
 
+    @wrappers.Request.application
+    def _serve_notifications(self, request):
+        """Serve a JSON runs of an experiment, specified with query param
+        `experiment`, with their nested data, tag, populated.
+
+        Runs returned are ordered by started time (aka first event time)
+        with empty times sorted last, and then ties are broken by
+        sorting on the run name. Tags are sorted by its name,
+        displayName, and lastly, inserted time.
+        """
+        # TODO: move this to note file
```

Comment:
Optional since I realize this is only temporary, but as a general practice we try to associate TODOs with either a username or (preferred) an issue, which makes it easier to follow up on them.  I'd consider just opening a single github issue for the overall notification center work, and then you can "cc" PRs to that issue, and use it for any TODOs like this.

E.g. to pick an example of my own (not that I've made much progress on that particular issue lately), there's [PR #4581](https://github.com/tensorflow/tensorboard/issues/4581) referenced here: https://github.com/tensorflow/tensorboard/blob/9bb887b334902e6b638ea3620d8203e05b700b78/tensorboard/summary/_writer.py#L34

Reply:
That makes sense. To simplify (all of those are coming up soon) I remove the data to make it empty